src/libaktualizr/storage/invstorage.h

```diff
@@ -184,7 +193,7 @@ class INvStorage {
   void importInstalledVersions(const boost::filesystem::path& base_path);
 
  protected:
-  const StorageConfig& config_;
+  const StorageConfig config_;
```

Comment:
What's wrong with the flaky reference?

Reply:
The configuration has to outlive the Storage object, that was annoying for the tests.